### PR TITLE
CONCF-682 | Internalization update

### PR DESCRIPTION
**Pull request by bognix**
https://wikia-inc.atlassian.net/browse/CONCF-682

### CODE REVIEW

File: front/locales/de/main.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "article-last-edit_plural_0":"Zuletzt diese Woche bearbeitet",
     "article-last-edit_plural_2":"Zuletzt vor __count__ Wochen bearbeitet",
     "article-categories-list-label":"Kategorien",
-    "article-top-contributors-label":"Top-Beitragende in diesem Artikel",
+    "article-top-contributors-label":"Top-Beitragende in diesem Wiki",
```
Comment (rogatty) [inline front/locales/de/main.json]: This is incorrect, previous version was correct.

**Comment by bognix** [comment]
Translation fixed also in crowdin

**Comment by rogatty** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**